rosdep/base.yaml

```diff
@@ -2901,6 +2905,7 @@ libvtk-java:
   gentoo: [sci-libs/vtk]
   slackware: [VTK]
   ubuntu: [libvtk-java]
+  opensuse: [vtk-java]
```

Comment:
OpenSuse does have support for versions like the other platforms: https://github.com/ros-infrastructure/rospkg/blob/master/src/rospkg/os_detect.py#L219